src/a4/domain/OpenSpace.java

```diff
@@ -1,5 +1,16 @@
 package a4.domain;
 
-public class OpenSpace {
+public class OpenSpace extends BoardSpace {
+	public OpenSpace() {
+	}
 
```

Comment:
Also, the way we're implementing OpenSpace, I don't think we really need a name on it since it should all work the same way.